### PR TITLE
Update extended-support-release.rst

### PR DESCRIPTION
**Pull request by Sayanta66**
Simpler writing with better clarity

### CODE REVIEW

File: source/upgrade/extended-support-release.rst [hunk 1]
```diff
@@ -12,7 +12,7 @@ During each monthly release, Mattermost backports high severity or high impact s
 What is the life cycle of an Extended Support Release?
 ------------------------------------------------------
 
-Mattermost provides an ESR when a significant number of new features and improvements have been added to the product, and those new features have had sufficient time to stabilize. A new ESR is released twice a year every January and July release. An ESR is supported for 9 months to provide customers enough time to test and upgrade to the next ESR.
+Mattermost provides an ESR when the product gets an addition of a significant number of new features and improvements, and those new features have had sufficient time to stabilize. A new ESR is released twice a year, every January and July release. An ESR is supported for 9 months to provide customers with enough time to test and upgrade to the next ESR.
```
Comment (justinegeffen) [inline source/upgrade/extended-support-release.rst:15]:
```suggestion
Mattermost provides an ESR when a significant number of new features and improvements have been added to the product, and have had sufficient time to stabilize. A new ESR is released twice a year, every January and July release. An ESR is supported for 9 months to provide customers with enough time to test and upgrade to the next ESR.
```

**Review by justinegeffen — COMMENTED** [review]
This is a great addition. 

The first suggested change didn't read as well as the original - but that is often the case. :) I did, however, review the whole paragraph and agree that it needed more clarity.

Let me know what you think - more than happy to discuss further.

**Comment by Sayanta66** [comment]
Yeah sure :)...I just think like writing in simple terms will help layman readers like me to have a look at the doc while browsing quickly

**Review by justinegeffen — APPROVED** [review]
A very helpful change, thank you!

**Comment by justinegeffen** [comment]
> Yeah sure :)...I just think like writing in simple terms will help layman readers like me to have a look at the doc while browsing quickly

I completely agree!

**Pull request merged**